### PR TITLE
Modify script to support webhook integration for Antithesis

### PR DESCRIPTION
**Pull request by Sam-Kramer**
## General
**Before this PR**:

**After this PR**:

==COMMIT_MSG==
Modify Antithesis publish script to support webhook integration 
==COMMIT_MSG==

**Priority**:

**Concerns / possible downsides (what feedback would you like?)**:

**Is documentation needed?**:

## Compatibility
**Does this PR create any API breaks (e.g. at the Java or HTTP layers) - if so, do we have compatibility?**:

**Does this PR change the persisted format of any data - if so, do we have forward and backward compatibility?**:

**The code in this PR may be part of a blue-green deploy. Can upgrades from previous versions safely coexist? (Consider restarts of blue or green nodes.)**:

**Does this PR rely on statements being true about other products at a deployment - if so, do we have correct product dependencies on these products (or other ways of verifying that these statements are true)?**:

**Does this PR need a schema migration?**

## Testing and Correctness
**What, if any, assumptions are made about the current state of the world? If they change over time, how will we find out?**:

**What was existing testing like? What have you done to improve it?**:

**If this PR contains complex concurrent or asynchronous code, is it correct? The onus is on the PR writer to demonstrate this.**:

**If this PR involves acquiring locks or other shared resources, how do we ensure that these are always released?**:

## Execution
**How would I tell this PR works in production? (Metrics, logs, etc.)**:

**Has the safety of all log arguments been decided correctly?**:

**Will this change significantly affect our spending on metrics or logs?**:

**How would I tell that this PR does not work in production? (monitors, etc.)**:

**If this PR does not work as expected, how do I fix that state? Would rollback be straightforward?**:

**If the above plan is more complex than “recall and rollback”, please tag the support PoC here (if it is the end of the week, tag both the current and next PoC)**:

## Scale
**Would this PR be expected to pose a risk at scale? Think of the shopping product at our largest stack.**:

**Would this PR be expected to perform a large number of database calls, and/or expensive database calls (e.g., row range scans, concurrent CAS)?**:

**Would this PR ever, with time and scale, become the wrong thing to do - and if so, how would we know that we need to do something differently?**:

## Development Process
**Where should we start reviewing?**:

**If this PR is in excess of 500 lines excluding versions lock-files, why does it not make sense to split it?**:

**Please tag any other people who should be aware of this PR**:
@jeremyk-91
@sverma30
@raiju

### CODE REVIEW

File: atlasdb-workload-server-distribution/build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.palantir.sls-java-service-distribution'
 apply plugin: 'com.palantir.external-publish-dist'
-apply from: "../gradle/docker.gradle"
 apply from: "../gradle/shared.gradle"
+apply from: "../gradle/docker.gradle"
```
Comment (Sam-Kramer) [inline atlasdb-workload-server-distribution/build.gradle:4]: This is very subtle, but very important. All of our images were `unspecified` as the gradle property hadn't been set yet. This get's set in `shared`!

Comment (jeremyk-91) [reply atlasdb-workload-server-distribution/build.gradle:4]: not for now: We might want to just have docker.gradle fail if it's included and the property isn't set - though definitely not for now!

File: scripts/circle-ci/publish-to-antithesis.sh [hunk 2]
```diff
@@ -20,11 +30,16 @@ docker pull palantirtechnologies/cassandra:2.2.18-1.112.0-rc5
...
+docker tag palantirtechnologies/timelock-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${1}
+docker push ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${1}
+
+docker tag palantirtechnologies/atlasdb-workload-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${1}
+docker push ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${1}
 
-docker tag palantirtechnologies/atlasdb-workload-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${VERSION}
-docker push ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-distribution:${VERSION}
+docker tag palantirtechnologies/atlasdb-workload-server-antithesis:${VERSION} ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-antithesis:${1}
+docker push ${ANTITHESIS_REPO_URL}/atlasdb-workload-server-antithesis:${1}
```
Comment (jeremyk-91) [inline scripts/circle-ci/publish-to-antithesis.sh:40]: question: Is there a difference between `$1` and `${1}`? Haven't seen this notation before, though pretty sure it works too

**Review by jeremyk-91 — APPROVED** [review]
👍

File: scripts/circle-ci/publish-to-antithesis.sh (hunk 2 above)
Comment (Sam-Kramer) [reply scripts/circle-ci/publish-to-antithesis.sh:40]: my bash knowledge is fairly poor tbh, but in this context I believe they are interchangeable? I think `${1}` is typically used for templating in strings, as `$1` would just render as one dollar. Not sure why I went with this way first time around!

**Pull request merged**